hw/ip_templates/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -193,15 +193,15 @@
 
     { struct:  "mubi4",
       type:    "uni",
-      name:    "hi_speed_sel",
-      act:     "req",
+      name:    "div_step_down_req",
+      act:     "rcv",
       package: "prim_mubi_pkg",
     },
 
     { struct:  "mubi4",
       type:    "uni",
-      name:    "div_step_down_req",
-      act:     "rcv",
+      name:    "hi_speed_sel",
+      act:     "req",
```

Comment:
These are accidental changes, right? If so, please remove them as they add noise to the diff